src/test/scala/org/broadinstitute/dsde/firecloud/integrationtest/ElasticSearchTrialDAOSpec.scala

```diff
@@ -0,0 +1,138 @@
+package org.broadinstitute.dsde.firecloud.integrationtest
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.FireCloudException
+import org.broadinstitute.dsde.firecloud.integrationtest.ESIntegrationSupport.{searchDAO, trialDAO}
+import org.broadinstitute.dsde.firecloud.model.WorkbenchUserInfo
+import org.broadinstitute.dsde.firecloud.model.Trial.TrialProject
+import org.broadinstitute.dsde.rawls.model.RawlsBillingProjectName
+import org.scalatest.{BeforeAndAfterAll, FreeSpec, Matchers}
+
+class ElasticSearchTrialDAOSpec extends FreeSpec with Matchers with BeforeAndAfterAll with LazyLogging  {
+
+  override def beforeAll = {
+    // using the delete from search dao, because we don't have recreate in trial dao.
+    searchDAO.recreateIndex()
+
+    // set up example data
+    logger.info("indexing fixtures ...")
+    ElasticSearchTrialDAOFixtures.fixtureProjects foreach { project => trialDAO.createProject(project.name)}
+
+    ElasticSearchTrialDAOFixtures.fixtureProjects collect {
+      case project if project.verified => trialDAO.verifyProject(project.name, verified=project.verified)
+    }
+    ElasticSearchTrialDAOFixtures.fixtureProjects collect {
+      case project if project.user.nonEmpty => trialDAO.claimProject(project.user.get)
+    }
+    logger.info("... fixtures indexed.")
+  }
+
+  override def afterAll = {
+    // using the delete from search dao, because we don't have recreate in trial dao.
+    searchDAO.deleteIndex()
+  }
+
+  "ElasticSearchTrialDAO" - {
```

Comment:
It would be nice to have some concurrency scenarios. On the other hand, I have no idea how to set that up, so I might be sending you on an impossible quest

Reply:
That was my thought process too. I bet I could figure out a way to do it, though.